app/assets/stylesheets/application.scss

```diff
@@ -54,6 +54,16 @@ html {
   }
 }
 
+#headerContainer {
```

Comment:
These style changes should be made inside `client/app/components/Header/Header.scss` instead, and should make these changes from inside of `Header` component in the React front-end!

Reply:
I originally tried doing just that, but found out that in order for css position: sticky to work, it needs an parent element to the Header component to whom it can 'stick to', and therefore some changes going up in the DOM tree are needed. I chose 'app/views/layouts/application.html.erb' and 'app/assets/stylesheets/application.scss' but it could be any ancestor of the Header component. What do you think?

Reply:
I think you can just setup this container element from inside the React component `client/app/components/Header/index.jsx` as a parent container, instead of doing it on the Rails view side.